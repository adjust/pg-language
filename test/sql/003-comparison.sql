SELECT 'ts' :: language <  'tt' :: language;

SELECT 'ts' :: language <= 'tt' :: language;

SELECT 'ts' :: language >  'tt' :: language;

SELECT 'tt' :: language >  'ts' :: language;

SELECT 'tt' :: language >= 'ts' :: language;

SELECT 'tt' :: language >  'tt' :: language;

SELECT 'tt' :: language >= 'tt' :: language;

SELECT 'tt' :: language =  'ts' :: language;

SELECT 'tt' :: language != 'ts' :: language;

SELECT 'tt' :: language != 'tt' :: language;

SELECT 'tt' :: language =  'tt' :: language;
