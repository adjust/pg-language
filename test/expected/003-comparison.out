SELECT 'ts' :: language <  'tt' :: language;
 ?column? 
----------
 t
(1 row)

SELECT 'ts' :: language <= 'tt' :: language;
 ?column? 
----------
 t
(1 row)

SELECT 'ts' :: language >  'tt' :: language;
 ?column? 
----------
 f
(1 row)

SELECT 'tt' :: language >  'ts' :: language;
 ?column? 
----------
 t
(1 row)

SELECT 'tt' :: language >= 'ts' :: language;
 ?column? 
----------
 t
(1 row)

SELECT 'tt' :: language >  'tt' :: language;
 ?column? 
----------
 f
(1 row)

SELECT 'tt' :: language >= 'tt' :: language;
 ?column? 
----------
 t
(1 row)

SELECT 'tt' :: language =  'ts' :: language;
 ?column? 
----------
 f
(1 row)

SELECT 'tt' :: language != 'ts' :: language;
 ?column? 
----------
 t
(1 row)

SELECT 'tt' :: language != 'tt' :: language;
 ?column? 
----------
 f
(1 row)

SELECT 'tt' :: language =  'tt' :: language;
 ?column? 
----------
 t
(1 row)

